test/extensions/filters/common/ext_authz/ext_authz_grpc_impl_test.cc

```diff
@@ -73,15 +71,13 @@ class ExtAuthzGrpcClientTest : public testing::TestWithParam<Params> {
   MockRequestCallbacks request_callbacks_;
   NiceMock<Event::MockDispatcher> dispatcher_;
   Tracing::MockSpan span_;
-  bool use_alpha_{};
   NiceMock<StreamInfo::MockStreamInfo> stream_info_;
   envoy::config::core::v3::ApiVersion api_version_;
 };
 
 INSTANTIATE_TEST_SUITE_P(Parameterized, ExtAuthzGrpcClientTest,
                          Values(Params(envoy::config::core::v3::ApiVersion::AUTO, false),
```

Comment:
I think you should update the above `Params` to only holding the versioning variations.

Reply:
Are you talking about removing `AUTO`?

Reply:
Nm, I see what you meant :)